.github/workflows/ci.yml

```diff
@@ -47,7 +47,7 @@ jobs:
             platform: android
             artifact-name: godot-cpp-android-arm64-release
             artifact-path: bin/libgodot-cpp.android.release.arm64.a
-            flags: arch=arm64
+            flags: ANDROID_NDK_ROOT=$ANDROID_NDK_LATEST_HOME arch=arm64
```

Comment:
Any reason to pass it via args instead of OS env?

Reply:
Long story should, I couldn't get github to play nice with env substitution, and the only way I found for it to work was adding an extra step.

Reply:
Yeah I remember struggling with that too in the past. Really obtuse way to pass env variables and badly documented, when it works at all.